main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -132,18 +172,40 @@ Refine.OpenProjectUI.prototype._addTagsListAnimation = function() {
     });
 };
 
-Refine.OpenProjectUI.prototype._fetchProjects = function() {
-    var self = this;
-    $.ajax({
-            type : 'GET',
-            url : "command/core/get-all-project-metadata",
-            dataType : 'json',
-            success : function(data) {
-                    self._renderProjects(data);
-                    self.resize();
-            },
-            data : {},
-            async : false
+Refine.OpenProjectUI.prototype._searchAnimation = function() {
+    var search = $('#searchIcon');
+    var form = $('.header-search-box');
+    var icon = $('.magnifying_glass');
+    search.click(function () {
+        if (form.is(':hidden'))
+        {
+            $("#tagsUl").hide()
+            $("#divInput").show()
+            icon.addClass("magnifying-glass-open")
+            form.show()
+        }
+        var widthFormOpen = Math.floor($('#right-panel-body').width() * 2 / 3);
+        form.animate({
+            'width': form.width() == widthFormOpen ? '0px' : widthFormOpen + "px"
+        }, 'fast', function () {
+            if (form.width() == 0) {
+                form.hide()
+                form.val('')
+                icon.removeClass("magnifying-glass-open")
+                $("#divInput").hide()
+                $("#tagsUl").show()
+            }
+        });
+    });
+};
+
+Refine.OpenProjectUI.prototype._searchInput = function() {
```

Comment:
can we make the search... case insensitive?

Reply:
Will case-insensitivity cause issues for those non-English users in the UI when they switch the default language to something other than English?  How will does that work in the search input?

Reply:
I did the changes to make it case insensitive. I am not sure about the issues for non-English users, can you clarify this for me? Thank you.